rfcs/0016-epoch-ledger-sync.md

```diff
@@ -0,0 +1,65 @@
+## Summary
```

Comment:
I just have some grammar nits but it’s too annoying to type on my phone. If someone copy edits I will approve. Otherwise will do when off the plane at some point.